sympy/printing/repr.py

```diff
@@ -48,7 +49,10 @@ def emptyPrinter(self, expr):
             return str(expr)
 
     def _print_Add(self, expr, order=None):
-        args = self._as_ordered_terms(expr, order=order)
+        if self.order == 'none':
```

Comment:
Could we make this `order == None`? Do we have precedent for string vs None elsewhere?

Reply:
There are a few different ordering options that each printer supports (such as `'old'` and `'lex'`) so that keeps in line with that convention. Having `order=None` means no ordering was provided rather than the "none" (aka don't reorder terms) ordering we are looking for. The overlap in terminology is confusing I do agree. The other printers use 'none' to denote no reordering of terms. We can revisit that terminology and sync the other printers accordingly but that would be backwards compatibility breaking and require to keep the old string lookup around.

Reply:
Yeah, 'none' is already a pre-existing thing. It's pretty confusing I agree. IIRC `None` means use the default (which I think is `'lex'`) whereas `'none'` means no ordering (all the orderings are strings).